pyproject.toml

```diff
@@ -27,7 +27,10 @@ rich = "^12.4.3"
 click = "8.1.2"
 importlib-metadata = "^4.11.3"
 typing-extensions = { version = "^4.0.0", python = "<3.8" }
-msgpack = "^1.0.3"
+
+# Dependencies below are required for devtools only
+aiohttp = { version = "^3.8.1", optional = true }
```

Comment:
@willmcgugan Do we move `click` into the `dev` extras too?

Reply:
I think so. Is it possible to install the `textual` command only with `dev` extras?

Reply:
I'm not sure I understand the question, but I don't believe you can install only the extras.

Reply:
We should only install the `textual` CLI when you `pip install textual[dev]`. Because we don't need it just to distribute apps. Is that possible?

Reply:
I don't believe so. Think we'd have to distribute the devtools as a separate package.